contractcourt/htlc_timeout_resolver.go

```diff
@@ -451,26 +453,6 @@ func (h *htlcTimeoutResolver) Resolve() (ContractResolver, error) {
 		return nil, h.claimCleanUp(commitSpend)
 	}
 
-	// At this point, the second-level transaction is sufficiently
```

Comment:
If we stop at this commit, yes, it would be delayed. Since a later commit breaks this into `Launch` and `Resolve`, when the first stage is resolved, it will call `checkpointStageOne` to send the resolution message.